test-adjuster/adjuster.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
I'm not sure if adjuster is the best name, because we don't know what it is adjusting.
Maybe frequency_adjuster or sth like that?